### ISSUE TITLE
Update-ScriptFileInfo adds blank lines between comment blocks each time it runs

### ISSUE THREAD

**Issue by jtstuedle** [PowerShell/PSResourceGet#347]
**Original Script File**
```
<#PSScriptInfo
....usual script info
#>

<#

    .Synopsis
    Blah

    .Description
    Blah
    ...
    ...
#>
```
**First Update-ScriptFileInfo run**
```powershell
Update-ScriptFileInfo -Path script.ps1 -Author blah -Description blah -Guid blah
```

**Updated Script File**
```
<#PSScriptInfo
....usual script info
#>


###Blank Line added here by update-scriptfileinfo every time it's run###
<#

    .Synopsis
    Blah

    .Description
    Blah
    ...
    ...
#>
```

**Comment by SydneyhSmith** [comment]
Thanks @jtstuedle this looks like it is potentially related to https://github.com/PowerShell/PowerShellGet/issues/316

**Comment by alerickson** [comment]
Resolved with PR #708
